common/src/IO/TextureLoader.cpp

```diff
@@ -58,6 +59,9 @@ namespace TrenchBroom {
             } else if (textureConfig.format.format == "idwal") {
                 TextureReader::PathSuffixNameStrategy nameStrategy(2, true);
                 return new IdWalTextureReader(nameStrategy, loadPalette(textureConfig));
+            } else if (textureConfig.format.format == "image") {
+                TextureReader::PathSuffixNameStrategy nameStrategy(3, true);
```

Comment:
I had to look up what the arguments to the PathSuffixNameStrategy constructor do.. maybe add a comment like `//generate texture name from path using the last 3 path components (last two directory levels + filename) and stripping the file extension`